### PR TITLE
kube-fzf: add page

### PR DESCRIPTION
**Pull request by NithyaNatarajan**
This PR adds tldr for `https://github.com/arunvelsriram/kube-fzf` repo

Signed-off-by: Nithya Natarajan <nithyanatarajn@gmail.com>

----

- [ x ] The page (if new), does not already exist in the repo.

- [ x ] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [ x ] The page has 8 or fewer examples.

- [ x ] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [ x ] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/kube-fzf.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# kube-fzf
+
+> Shell commands for command line fuzzy searching of Kubernetes Pods.
+
+- To get pod details (searching from current namespace):
```
Comment (mfrw) [inline pages/common/kube-fzf.md]: How about:
```suggestion
- Get pod details (searching from current namespace):
```

File: pages/common/kube-fzf.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+`findpod`
+
+- To get pod details (searching from all namespaces):
```
Comment (mfrw) [inline pages/common/kube-fzf.md]:
```suggestion
- Get pod details (searching from all namespaces):
```

File: pages/common/kube-fzf.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+`findpod -a`
+
+- To describe a pod:
```
Comment (mfrw) [inline pages/common/kube-fzf.md]:
```suggestion
- Describe a pod:
```

File: pages/common/kube-fzf.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+`describepod`
+
+- To tail pod logs:
```
Comment (mfrw) [inline pages/common/kube-fzf.md]:
```suggestion
- Tail pod logs:
```

File: pages/common/kube-fzf.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+`tailpod`
+
+- To exec into a pod's container:
```
Comment (mfrw) [inline pages/common/kube-fzf.md]:
```suggestion
- Exec into a pod's container:
```

File: pages/common/kube-fzf.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+`findpod -a`
+
+- To describe a pod:
+
+`describepod`
+
+- To tail pod logs:
+
+`tailpod`
```
Comment (mfrw) [inline pages/common/kube-fzf.md]:
```suggestion
`tailpod {{podname}}`
```

File: pages/common/kube-fzf.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+- To exec into a pod's container:
+
+`execpod`
+
+- To port-forward a pod:
```
Comment (mfrw) [inline pages/common/kube-fzf.md]:
```suggestion
- Port-forward a pod:
```

**Comment by mfrw** [comment]
@agnivade it is actually a bunch of shell scripts. kube-fzf in itself is the repo.

**Comment by NithyaNatarajan** [comment]
@mfrw and @agnivade, will accommodate the suggestions.

yes @agnivade, it is just a name for a bunch of shell scripts. Let me know if I should mention it somewhere on tldr page if it is confusing.

File: pages/common/kube-fzf.md (hunk 6 above)
Comment (NithyaNatarajan) [reply pages/common/kube-fzf.md]: `tailpod` will give a fuzzy search option through which we can select a pod name. `tailpod` does not take a pod name.

Comment (mfrw) [reply pages/common/kube-fzf.md]: yep....

**Comment by sbrl** [comment]
Generally speaking, each command should have its own page, @NithyaNatarajan. If there are a bunch of related commands, we usually add a second description line that says something like "see also....."

File: pages/common/kube-fzf.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+- Tail pod logs:
+
+`tailpod`
+
+- Exec into a pod's container:
+
+`execpod`
+
+- Port-forward a pod:
```
Comment (agnivade) [inline pages/common/kube-fzf.md]: To what port no. ?

File: pages/common/kube-fzf.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
+# kube-fzf
+
+> Shell commands for command line fuzzy searching of Kubernetes Pods.
+
+- Get pod details (searching from current namespace):
```
Comment (agnivade) [inline pages/common/kube-fzf.md]: We can drop "searching"

File: pages/common/kube-fzf.md [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+
+`findpod`
+
+- Get pod details (searching from all namespaces):
```
Comment (agnivade) [inline pages/common/kube-fzf.md]: Same as above

**Pull request merged**